ext/storage_sources/gcp_store/gcp_connection.cpp

```diff
@@ -37,6 +37,16 @@ gcp_connection::gcp_connection(const std::string &bucket_name, const std::string
     : _gcp_client(google::cloud::storage::Client()), _bucket_name(bucket_name),
       _object_prefix(prefix)
 {
+    /* StatusOr either contains a usable BucketMetadata value or a Status object explaining why
+     * a BucketMetadata is not present.
+     * Its validity will be checked using a conversion to bool.
```

Comment:
I would consider rewording this to something like `StatusOr either contains a usable BucketMetadata value or a Status object explaining why the value is not present. The value's validity is checked by StatusOr::ok().`

Do we want to use `//` comments here?

Reply:
I've changed it to `//` comment here: [9d94f39](https://github.com/wiredtiger/wiredtiger/pull/8737/commits/9d94f3902f16dbc52ad03b514c6efcceb6b6f14f) :)